pyanaconda/bootloader.py

```diff
@@ -2009,7 +2009,7 @@ class Yaboot(YabootBase):
 
     # stage2 device requirements
     stage2_device_types = ["partition", "mdarray"]
-    stage2_device_raid_levels = [raid.RAID1]
+    stage2_raid_levels = [raid.RAID1]
```

Comment:
Wow this may fix some ugly issues or introduce them?

Reply:
This may enable installation of the Yaboot boot loader on RAID1 again. The attribute was renamed in a commit cc17c1b019d17400cccc1e006b5314adcb531537, so this is clearly a bug.

Reply:
Ok thanks!